components/builder-protocol/protocols/originsrv.proto

```diff
@@ -341,8 +341,7 @@ message OriginProject {
   optional uint64 owner_id = 7;
   optional string vcs_type = 8;
   optional string vcs_data = 9;
-  optional string vcs_auth_token = 10;
-  optional string vcs_username = 11;
+  optional uint64 vcs_installation_id = 10;
```

Comment:
We can't reuse field numbers so you'd want to make this `12`

Reply:
It's intended that this protocol message is polymorphic across many different types of VCS authentication. I don't think I'd suggest the extra work right now since we aren't trying to support more than one VCS, but installation_id is a thing specific to GitHub. Go with what you got but we'll maybe want to revisit this later

Reply:
yeah I think we will need to make deeper changes to make the polymorphism more natural but its really too early until we have at least 2 VCSs.

Reply:
is it ok to break back compat here since we have not deployed the above 2 fields to prod?

Reply:
No, we need to consider acceptance production as well. Theoretically it's ok because it would only cause a super small outage lasting less then a few minutes, but it's best to just get in the Habitat of versioning the protocols and stored procedures properly